tests/testthat/test_dir/test-skip.r

```diff
@@ -0,0 +1,6 @@
+context("Skip")
```

Comment:
Done, though I noticed that "Bare" and "Basic" in test_dir are also capitalized.